pkg/k8shandler/secret_test.go

```diff
@@ -0,0 +1,265 @@
+package k8shandler
+
+import (
+	"reflect"
+	"testing"
+
+	logging "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1"
+	rbac "k8s.io/api/rbac/v1"
+
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+)
+
+func TestClusterLoggingRequest_CreateOrUpdateSecretNoop(t *testing.T) {
+	clusterRoleBinding := &rbac.ClusterRoleBinding{}
+
+	clusterLoggingRequest := &ClusterLoggingRequest{
+		client:  fake.NewFakeClient(clusterRoleBinding),
+		cluster: &logging.ClusterLogging{},
+	}
+
+	// mimic ca.crt
+	caCrtStr := `-----BEGIN CERTIFICATE-----
```

Comment:
Per @richm other comments, the content here is likely immaterial given we are just testing if we can stuff a value into a secret and its updated (or not) properly

Reply:
I like the fact that it sort of looks like a cert value and is multiline.